kalite/distributed/templates/distributed/base.html

```diff
@@ -202,29 +202,31 @@
                             {% endif %}
 
                             {# Manage for admins => zone control panel #}
-                            <li class="{% block manage_active_admins %}{% endblock manage_active_admins %}">
-                              <a href="{% url 'zone_redirect' %}" class="admin-only">
-                                {% trans "Manage" %}
-                              </a>
-                            </li>
                             <!-- End for logged in admins -->
 
                             <!-- For logged in teachers -->
                             {# Teach for teachers => coachreports #}
-                            <li class="{% block teach_active_teachers %}{% endblock teach_active_teachers %}">
-                              <a href="{% url 'tabular_view' %}" id="nav_coachreports" class="teacher-only">
-                                {% trans "Teach" %}
-                              </a>
-                            </li>
+{#                            <li class="{% block teach_active_teachers %}{% endblock teach_active_teachers %}">#}
+{#                              <a href="{% url 'tabular_view' %}" id="nav_coachreports" class="teacher-only">#}
+{#                                {% trans "Teach" %}#}
+{#                              </a>#}
+{#                            </li>#}
```

Comment:
These were commented out, right? Wouldn't it be good to just delete this rather than keep them around?